client/elements/sc-page-static.html

```diff
@@ -182,30 +226,64 @@ <h3>Page not found</h3>
   </template>
 
   <script>
-      class SCPageStatic extends ReduxMixin(Polymer.Element) {
-          static get is() {
-              return 'sc-page-static';
-          }
+    class SCPageStatic extends ReduxMixin(Polymer.Element) {
+      static get is() {
+          return 'sc-page-static';
+      }
 
-          static get properties() {
-              return {
-                  selectedPage: {
-                      type: String,
-                      observer: '_changeView',
-                      statePath: 'currentRoute.name'
-                  },
-                  pages: {
-                      type: String,
-                      value: 'HOME'
-                  }
+      static get properties() {
+          return {
+              selectedPage: {
+                  type: String,
+                  observer: '_changeView',
+                  statePath: 'currentRoute.name'
+              },
+              pages: {
+                  type: String,
+                  value: 'HOME'
               }
           }
+      }
 
-          _changeView() {
-              this.$.pages.selected = this.selectedPage;
+      ready() {
+          super.ready();
+          this._addLinkClickHandlers();
+      }
+
+      _selectNavbarLink() {
+          const tabs = this.$.nav_tabs.querySelectorAll('paper-tab');
+          for (let i = 0; i < tabs.length; i++) {
+              if (tabs[i].dataset.name === this.selectedPage) {
+                  this.$.nav_tabs.selectIndex(i);
+                  return;
+              }
           }
+          this.$.nav_tabs.selectIndex(-1);
+      }
+
+      _addLinkClickHandlers() {
+          this.$.nav_tabs.addEventListener('selected-item-changed', e => {
+              let target = e.detail.value;
+              if (!target) {
+                  return;
+              }
+              target.querySelector('.hidden-link').click();
+          });
+      }
+
+      connectedCallback() {
+          super.connectedCallback();
+          this.shadowRoot.querySelectorAll('.link-text').forEach(link => {
+              link.style.width = window.getComputedStyle(link).width;
```

Comment:
what does this line do?

Reply:
It keeps bar from moving all the items when one gets bolded

Reply:
seems like something that should be done with CSS. Changing the width in JS causes reflow, which we want to avoid.